device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -526,6 +563,18 @@ public void sendMessages()
                     //Codes_SRS_IOTHUBTRANSPORT_34_044: [This function continue to dequeue packets saved in the waiting
                     // queue and send them until connection status isn't CONNECTED or until 10 messages have been sent]
                     sendPacket(packet);
+
+                    try {
+                        String messageId = message.getCorrelationId();
+
+                        if (messageId != null && correlationCallbacks.containsKey(messageId))
+                        {
```

Comment:
inconsistent bracing style

Reply:
Yeah got with Tim about that, was bit by the auto format